.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
This is going to fail at the moment, but I will work on a fix later today. Just getting this started.

Reply:
Sure, understood :-).

Reply:
Sorry for this. I believe this is my fault, hence I fixed it. I do not use docker much but had to use it locally to test things with 3.11. I believe this is now all good.